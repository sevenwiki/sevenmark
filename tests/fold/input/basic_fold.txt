{{{#fold
[[first content]]
[[second content]]
}}}