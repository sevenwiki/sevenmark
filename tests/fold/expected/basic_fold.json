[
  {
    "FoldElement": {
      "location": {
        "start": 0,
        "end": 49
      },
      "parameters": {},
      "content": [
        {
          "parameters": {},
          "content": [
            {
              "Text": {
                "location": {
                  "start": 11,
                  "end": 24
                },
                "content": "first content"
              }
            }
          ]
        },
        {
          "parameters": {},
          "content": [
            {
              "Text": {
                "location": {
                  "start": 29,
                  "end": 43
                },
                "content": "second content"
              }
            }
          ]
        }
      ]
    }
  }
]